tests/pytest.ini

```diff
@@ -12,4 +12,5 @@ markers:
     topology: specify which topology testcase can be executed on: (t0, t1, ptf, etc)
     platform: specify which platform testcase can be executed on: (physical, virtual, broadcom, mellanox, etc)
 
-log_format: %(asctime)s %(levelname)s %(filename)s:%(funcName)s:%(lineno)d: %(message)s
+log_cli_format: %(asctime)s %(levelname)s %(filename)s:%(funcName)s:%(lineno)d: %(message)s
```

Comment:
https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-image-201911-test/155/artifact/sonic-mgmt/tests/logs/1vlan/bgp/test_bgp_gr_helper.log

can you add data in the timestamp?